tools/m2ctx/m2ctx.py

```diff
@@ -79,14 +79,15 @@ def write_header(path: Path):
     path.write_text("\n".join(files), encoding="utf-8")
 
 
-def try_import(c_command: List[str]):
+def try_import(c_command: List[str], args_quiet):
     try:
         out_text = subprocess.check_output(c_command, cwd=root, encoding="utf8")
     except subprocess.CalledProcessError as err:
+        err_output = "" if args_quiet else err.output
         print(
             "Failed to preprocess input file, when running command:\n"
             + " ".join(c_command)
-            + f"\n\n{err.output}",
+            + f"\n\n{err_output}",
```

Comment:
The trailing newlines are a bit messy. Could either construct an `output` variable, or just do more than one `print` statement with `if not quiet`.